helloworld/notebooks/OpHousingPrices.ipynb

```diff
@@ -0,0 +1,386 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val yrSold = FeatureBuilder.Integral[HousingPrices].extract(x =>\n",
+    "      {\n",
+    "        var y =  2019 - x.yrSold\n",
+    "        y.toIntegral\n",
+    "      }\n",
+    "    ).asPredictor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " val saleType = FeatureBuilder.Integral[HousingPrices].extract(x =>\n",
+    "      {\n",
+    "        val y = x.saleType\n",
+    "        val z = saleTypeEncoder.get(y)\n",
+    "        z.toIntegral\n",
+    "      }\n",
+    "    ).asPredictor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val saleConditionEncoder = Map(\"Abnorml\" -> 1, \"AdjLand\" -> 2, \"Alloca\" -> 3, \"Family\" -> 4,\n",
+    "      \"Normal\" -> 5, \"Partial\" -> 6 )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val saleCondition = FeatureBuilder.Integral[HousingPrices].extract(x =>\n",
+    "    {\n",
+    "      val y = x.saleCondition\n",
+    "      val z = saleConditionEncoder.get(y)\n",
+    "      z.toIntegral\n",
+    "    }\n",
+    "    ).asPredictor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val salePrice = FeatureBuilder.RealNN[HousingPrices].extract(_.salePrice.toRealNN).asResponse"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " val trainFilePath = \"/home/beakerx/helloworld/src/main/resources/HousingPricesDataset/train_lf_la_ls_ys_st_sc.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a training data reader from the `trainFilePath` using `DataReaders.Simple`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val trainDataReader = DataReaders.Simple.csvCase[HousingPrices](\n",
+    "      path = Option(trainFilePath)\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Create a feature sequence and transmogrify it**\n",
+    "\n",
+    "The `.transmogrify()` shortcut is a special AutoML Estimator that applies a default set of transformations to all the specified inputs and combines them into a single vector. This is in essence the automatic feature engineering Stage of TransmogrifAI. This stage can be discarded in favor of hand-tuned feature engineering and manual vector creation followed by combination using the VectorsCombiner Transformer (short-hand Seq(....).combine()) if the user desires to have complete control over feature engineering.\n",
```

Comment:
it is a Transmografai shortcut to many estimators

Reply:
changed